package.json

```diff
@@ -46,7 +47,6 @@
     "@wordpress/compose": "^3.23.0",
     "@wordpress/data": "^4.26.0",
     "@wordpress/element": "^2.19.0",
-    "@wordpress/i18n": "^3.17.0",
```

Comment:
This should be kept for now as it would be directly used by activation-notice and story-embed-block.